### PR TITLE
【TEST】Fixed proper comment

### PR DESCRIPTION
**Pull request by braveek1**

### CODE REVIEW

File: routes/index.js [hunk 1]
```diff
@@ -2,7 +2,8 @@ var express = require("express")
 var router = express.Router()
 
 /* GET home page. */
-// test4
+// introduce myself
```
Comment (braveek1) [inline routes/index.js:5]: OK

**Review by braveek1 — COMMENTED** [review]
checked

**Review by braveek1 — COMMENTED** [review]
checked

**Review by braveek1 — COMMENTED** [review]
Check

File: routes/index.js (hunk 1 above)
Comment (leehanbi) [reply routes/index.js:5]: // test4
// introduce myself
remove comment plz

**Pull request merged**